src/mobile/config.coffee

```diff
@@ -30,6 +30,8 @@ module.exports =
   GEMINI_CLOUDFRONT_URL: 'https://d7hftxdivxxvm.cloudfront.net'
   GOOGLE_ANALYTICS_ID: 'UA-12450662-6'
   GOOGLE_MAPS_API_KEY: null
+  INTERCOM_APP_ID: null
+  INTERCOM_ENABLED: false
```

Comment:
Not sure if this is needed because _I think_ the partnerships page is responsive and we don't have a separate mobile version.